CHANGES_UNRELEASED.md

```diff
@@ -3,3 +3,9 @@
 # Unreleased Changes
 
 [Full Changelog](https://github.com/mozilla/application-services/compare/v0.58.2...master)
+
+## Viaduct
+
+### Breaking changes
+
+- The `include_cookies` setting is not supported anymore. ([#3096](https://github.com/mozilla/application-services/pull/3096))
```

Comment:
nit: I think it would be worth noting explicitly that they're always disabled (the current changelog implies that you can no longer control it, but doesn't say what the default is, you have to dig into the readme for that).